sdk/messaging_servicebus/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "azure_messaging_servicebus"
-version = "0.2.0"
+version = "0.3.0"
```

Comment:
You can revert this back to `0.2.0`. There is only a `0.1.0` published. https://crates.io/crates/azure_messaging_servicebus/versions

Reply:
done ✅